### ISSUE TITLE
Export custom functions and collations as a file

### ISSUE THREAD

**Issue by prasy-loyola** [pawelsalawa/sqlitestudio#3531]
Great Job!
I understand that the custom functions and collations work only with SQLiteStudio.
Will it be possible to export the custom functions and collations written in QtScript as a file so that they can be shared with others who can import into their SQLiteStudio and use them.

**Comment by pawelsalawa** [comment]
Good idea! I'll treat this issue as enhancement proposal for the same.
